src/scss/_informer.scss

```diff
@@ -45,3 +45,356 @@
     line-height: 40px;
   }
 }
+
+.UppyInformer span {
+  line-height: 12px;
+  width: 13px;
+  height: 13px;
+  display: inline-block;
+  vertical-align: middle;
+  background-color: $color-white;
+  border-radius: 50%;
+  position: relative;
+  top: -1px;
+  left: 5px;
+  font-size: 10px;
+  margin-left: -1px;
+}
+
+.UppyInformer span:after {
+  line-height: 1.3;
+  word-wrap: break-word;
+}
+
+// balloon
+// https://kazzkiq.github.io/balloon.css/
```

Comment:
Minor, but perhaps this could be in a separate file?

Reply:
I was thinking of removing the extras and just making it part of the Informer, but maybe balloons could be useful elsewhere.

Reply:
yeah i was suggesting it mostly because balloon.css is a dependency, but if the unnecessary stuff is removed I'd be OK with having it in this file as well.